src/utils/currency.ts

```diff
@@ -52,57 +46,51 @@ export function roundLastNDigits(n: number, x: BN | Big | string): string {
     return bigNumber.div(new BN(power)).mul(new BN(power)).toString();
 }
 
-export function newMonetaryAmount<U extends CurrencyUnit>(
-    amount: BigSource,
-    currency: Currency<U>,
-    base = false
-): MonetaryAmount<Currency<U>, U> {
-    const unit = base ? currency.base : currency.rawBase;
-    return new MonetaryAmount<Currency<U>, U>(currency, amount, unit);
+export function atomicToBaseAmount(atomicAmount: BigSource, currency: Currency): Big {
+    return new Big(atomicAmount).div(new Big(10).pow(currency.decimals));
+}
+
+export function newMonetaryAmount(amount: BigSource, currency: Currency, base = false): MonetaryAmount<Currency> {
```

Comment:
I like that you kept this for backward-compatiblity, even though it's no longer required :) Maybe add a `@deprecated` to inform of the intended usage with the simplified `MonetaryAmount`